### DISCUSSION TITLE
Out of range for changeset len 25336

### DISCUSSION THREAD

**Discussion by ilyaaay** [helix-editor/helix#10837] [Troubleshooting]
### Summary

I often encounter this problem during work.

### Helix log

thread 'main' panicked at helix-core/src/transaction.rs:478:9:
Positions [(25353, After), (25354, Before)] are out of range for changeset len 25336!
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

### Platform

linux (Manjaro)

### Terminal Emulator

kitty

### Installation Method

yay (AUR)

### Helix Version

helix 24.3

**Comment by pascalkuthe** [comment]
This is not actionable without a backtrace or reproduction case. Thus can be caused by anything.

Do you have any insertmode keybindign like c-s to save in insertmode? Binding normal mode bindings in insertmode is unsupported and causes crashes like this

**Reply by ilyaaay** [reply]
Thanks for reply! No, i dont have a bindings at all...
